packages/@webex/plugin-meetings/src/member/index.ts

```diff
@@ -207,7 +212,7 @@ export default class Member {
      * @public
      * @memberof Member
      */
-    this.associatedUser = null;
+    this.associatedUsers = new Set();
```

Comment:
new Set<>

Reply:
added, although I'm not sure if it's needed because of the type declaration earlier already stating that